src/contracts/models/factories/GnosisSafe__factory.ts

```diff
@@ -0,0 +1,71 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
I would say that we should remove all these autogenerated files and ignore them with `.gitignore`